### PR TITLE
Add minutes from the meeting on 20210923.

### PR DESCRIPTION
**Pull request by johanfforsberg**

### CODE REVIEW

File: 20210923/MINUTES.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  - A GUI is developed, and being taken into production
+
+2. Urgent user problems/issues - Round table
+   - MAX IV
+     - [X] Ctrl-C crash at MAX IV related to reading State. Henrik made a MR: https://github.com/sardana-org/sardana/pull/1695
+     - [ ] Issue at MAX IV related to ADLINK controller from ALBA, related to the State. Seems this is only in use at some specific place at ALBA. Zibi will check with them.
+   - ALBA
+     - [ ] Exotic problem with stateless subscriptions in PyTango that can cause issues in "fast dummy motor controller". Happens when taurus tries to subscribe but fails. Can be caused by changing event config. No decision on how to proceed.
+     - [ ] `self.<macro name>` exceptions are uninformative.
+     - [ ] Migration of final sardana 2 beamlines -> 3. Scans using two different pools together experience hangs. Perhaps some cases are related to state attribute polling.
```
Comment (reszelaz) [inline 20210923/MINUTES.md]:
```suggestion
     - Migration of final sardana 2 beamlines -> 3. 
     - [ ] Scans using two different pools together experience hangs. Perhaps some cases are related to state attribute polling. 
```
I would keep it as two separe points. And about migration, there is no action to be taken.

File: 20210923/MINUTES.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            - Demo of "sherlock" by Henrik/MAX IV https://gitlab.com/MaxIV/app-maxiv-sherlock
+              + We could add checking of attribute event settings 
+              + Should be listed in the Sardana plugins catalog!
+        - [ ] Hang scan due to the counter/timer controller timeout in StateOne() 
+            - How to deal with exceptions in the controllers: Review the docs?
+            - Proposal for updating docs: https://github.com/sardana-org/sardana/issues/1693
+            - Henrik/MAXIV will look into adding functionality to the IcePap controller to show motor OFF state.
+        
+     - New:
+       - GUIs sardanaeditor and "showscan offline" will likely be removed from sardana as a result of low interest in the sardana user survey. Zibi will make a PR.
```
Comment (reszelaz) [inline 20210923/MINUTES.md]:
```suggestion
       - [ ] GUIs sardanaeditor and "showscan offline" will likely be removed from sardana as a result of low interest in the sardana user survey. Zibi will make a PR.
```

I would treat it as a TODO action.

File: 20210923/MINUTES.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  - Mainly Carlos and Benjamin will perform the migration 
+    - Carlos will make announcement, leaving at least two weeks for users to respond
+    - Bunch of other things to also migrate
+  - Documentation build is currently broken anyway
+   
+6. Sardana release 3.2 (Jul21 milestone)
+  - Need to wait for Taurus to make a release ~2 weeks.
+  - Solaris and DESY are supposed to handle this release
+  - The bug squashing party merged an issue that also requires some discussion
+    - Stored start-time should be when the scan starts, not when the macro starts (Henrik)
```
Comment (reszelaz) [inline 20210923/MINUTES.md]:
```suggestion
    - [ ] Stored start-time should be when the scan starts, not when the macro starts (Henrik)
```
I would treat it as a TODO action

**Review by reszelaz — APPROVED** [review]
Many thanks @johanfforsberg ! Very nice minutes!

And of course many thanks to @teresanunez for the Sardana Configuration Tool presentation!

And it was nice to see new faces: @saidaHF and @marc2332 and to see again @dschick!

**Review by teresanunez — APPROVED** [review]

**Comment by reszelaz** [comment]
Hi all,
I remember that we agreed on the next meeting date, but I'm not sure about the day. Was it 28th of October at 14:00? It will be SOLARIS turn to organize the meeting this time. 
Cheers

**Comment by teresanunez** [comment]
Hi Zibi, 
I also do not remember it, and I do not see it in the minutes. 
For me it would be fine that date. 

Best Regards, 
Teresa

**Comment by reszelaz** [comment]
I let myself commit the suggestions I've made some time ago. I will now proceed to merge the minutes.

**Pull request merged**